### ISSUE TITLE
Remove code from README

### ISSUE THREAD

**Issue by raq929** [ga-wdi-boston/js-basics#49]
All code examples should be in files that are linked to from the README.

**Comment by payne-chris-r** [comment]
I disagree. This needs to be discussed and we need to make a firm decision on this.

**Comment by payne-chris-r** [comment]
Per conversations with @gaand and @jrhorn424 we need to leave code in READMEs for now until we figure out a better way to maintain code, but still display it in the README. For now, we leave the READMEs as is, and do our best to make sure that the code we have in there is correct, opening issues and resolving if it is not.

**Comment by gaand** [comment]
https://github.com/ga-wdi-boston/node-template/blob/add-code-blocks/bin/add-code-block.js
